src/train.py

```diff
@@ -237,11 +237,11 @@ def supervised_step(model, batch, args, use_dis_loss=False):
     mel_aug = specaugment(mel, mel_len)
     if use_dis_loss:
         pre_pred, post_pred, stop_pred, stop_lens, t_hid = model.tts(text, text_len, mel, mel_len, ret_enc_hid=use_dis_loss)
-        t_d_loss = discriminator_hidden_to_loss(model, t_hid, 'speech', freeze_discriminator=True)
+        t_d_loss = discriminator_hidden_to_loss(model, t_hid, text_len, 'speech', args.model_type, freeze_discriminator=True)
```

Comment:
Please check that these make sense @billyang98 - essentially these should be the ground truth lengths of the encoded state so that we know which encoder outputs correspond to padding.